sm/engine/dataset_manager.py

```diff
@@ -265,3 +270,9 @@ def add_optical_image(self, ds, optical_scan, transform, zoom_levels=[1, 2, 4, 8
         self._add_raw_optical_image(ds, optical_scan, transform)
         self._add_zoom_optical_images(ds, optical_scan, transform, zoom_levels)
 
+    def del_optical_image(self, ds, **kwargs):
+        "Deletes Raw Image from DB"
```

Comment:
To add a docstring use three double quotes https://www.python.org/dev/peps/pep-0257/